### PR TITLE
enabled linters and fixes for: misspell, predeclared, unconvert, ineffassign, gosimple, govet

### PR DESCRIPTION
**Pull request by mmetc**

### CODE REVIEW

File: cmd/crowdsec/output.go [hunk 1]
```diff
@@ -125,8 +125,6 @@ LOOP:
 			if len(cache) > 0 {
 				cacheMutex.Lock()
 				cachecopy := cache
-				newcache := make([]types.RuntimeAlert, 0)
-				cache = newcache
```
Comment (buixor) [inline cmd/crowdsec/output.go:129]: suppressing this is going to skip "emptying" the cache no ?

Comment (mmetc) [reply cmd/crowdsec/output.go:129]: If I'm not mistaken, we are exiting the function after this block (break LOOP), and cache disappears

**Review by buixor — CHANGES_REQUESTED** [review]
see my comment, but overall lgtm

**Review by buixor — APPROVED** [review]
lgtm

**Pull request merged**